src/helpers/chain-providers.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FlatProviders, Providers } from '../core/flat-providers';
+import { ProviderComponent, ProviderComponentProps } from '../types/provider';
+
+export function useChainProviders() {
```

Comment:
@davelosert For now, it's named 'useChainProviders' but that's one of the names that came to my mind, what options do you have?

Reply:
I wouldn't name it `use` as that is the name for `hooks` and `custom hooks` in React, so that might be a little pitfall.

How about just something like `createProviders`, `providerBuilder` or simply `chainProviders`?